src/components/ha-selector/ha-selector-select.ts

```diff
@@ -26,25 +31,60 @@ export class HaSelectSelector extends LitElement {
   @property({ type: Boolean }) public required = true;
 
   protected render() {
-    if (this.required && this.selector.select.options!.length < 6) {
```

Comment:
Should we render with checkboxes ?

Reply:
if few choices.

Reply:
maybe 🤷

Reply:
Yeah, I think adding checkboxes is a great idea